qmetaobject_impl/src/qobject_impl.rs

```diff
@@ -1153,7 +1153,8 @@ fn is_valid_repr_attribute(attribute: &syn::Attribute) -> bool {
             if list.path.is_ident("repr") && list.nested.len() == 1 {
                 match &list.nested[0] {
                     syn::NestedMeta::Meta(syn::Meta::Path(word)) => {
-                        const ACCEPTABLE_TYPES: &[&str] = &["u8", "u16", "u32", "i8", "i16", "i32"];
+                        const ACCEPTABLE_TYPES: &[&str] =
```

Comment:
While you are at it, can you please also rename this to `ACCEPTABLE_REPRESENTATIONS` or `*_LAYOUTS`?

https://doc.rust-lang.org/nomicon/other-reprs.html